rxbinding-support-recyclerview/src/androidTest/AndroidManifest.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<manifest package="com.jakewharton.rxbinding.support.v7.recyclerview"
+          xmlns:android="http://schemas.android.com/apk/res/android">
+  <application>
+    <activity android:name="com.jakewharton.rxbinding.widget.RxAdapterViewTestActivity"/>
```

Comment:
Derp, that's what I get for blindly trusting autocomplete. Odd that it worked fine though